<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Budget</class>
 <widget class="QDialog" name="Budget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Budget</string>
  </property>
  <widget class="QWidget" name="Fback" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>720</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
	background-color: rgb(36, 36, 36);
}</string>
   </property>
   <widget class="QWidget" name="Fback1" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>1221</width>
      <height>661</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
	background-color: rgb(45, 45, 45);
}</string>
    </property>
    <widget class="QWidget" name="Fback2" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>1171</width>
       <height>611</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{
	background-color: rgb(55, 55, 55);
	border-radius:20px;
}</string>
     </property>
     <widget class="QWidget" name="Fback3" native="true">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>1121</width>
        <height>551</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget{
	background-color: rgb(65, 65, 65);
	border-radius:15px;
}</string>
      </property>
      <widget class="QWidget" name="Fback4" native="true">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>501</width>
         <height>471</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget{
	
	background-color:rgb(88, 88, 88);
	border-radius:10px;
}</string>
       </property>
       <widget class="QComboBox" name="cat_Budget">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>140</y>
          <width>271</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Cascadia Code</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QComboBox{
	background-color:rgb(255, 131, 133);
	border-radius:20px;
	padding-left:10px;
	color:black;
}</string>
        </property>
        <item>
         <property name="text">
          <string>Food</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Entertainment</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Traveling</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Clothing</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Education</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>University Fees</string>
         </property>
        </item>
       </widget>
       <widget class="QPushButton" name="btnBudget">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>330</y>
          <width>121</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Rockwell</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color:rgb(255, 213, 215);
	border-radius:25px;
	border-color: rgb(53, 69, 93);
	
	color: rgb(0, 0, 0);
}</string>
        </property>
        <property name="text">
         <string>Set Budget</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_date">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>200</y>
          <width>161</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Rockwell</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
	color:rgb(255, 213, 215);
}</string>
        </property>
        <property name="text">
         <string>Month</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_cat">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>110</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Rockwell</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
	color:rgb(255, 213, 215);
}</string>
        </property>
        <property name="text">
         <string>Category</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_amount">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>300</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Rockwell</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
	color:rgb(255, 213, 215);
}</string>
        </property>
        <property name="text">
         <string>Amount</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnClearALL">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>230</y>
          <width>121</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Rockwell</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color:rgb(255, 213, 215);
	border-radius:25px;
	border-color: rgb(53, 69, 93);
	
	color: rgb(0, 0, 0);
}</string>
        </property>
        <property name="text">
         <string>Clear All</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnSelected">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>140</y>
          <width>131</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Rockwell</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color:rgb(255, 213, 215);
	border-radius:25px;
	border-color: rgb(53, 69, 93);
	
	color: rgb(0, 0, 0);
}</string>
        </property>
        <property name="text">
         <string>Clear Selected</string>
        </property>
       </widget>
       <widget class="QLabel" name="txtBudget">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>10</y>
          <width>381</width>
          <height>91</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Poor Richard</family>
          <pointsize>36</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
	color:rgb(255, 213, 215);
}</string>
        </property>
        <property name="text">
         <string>Set Budget</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnBack_2">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>410</y>
          <width>121</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color:rgb(152, 96, 100);
	border-radius:25px;
	border-color: rgb(53, 69, 93);
	
	color: gold;
}</string>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
       <widget class="QDateEdit" name="month">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>230</y>
          <width>271</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Cascadia Code</family>
         </font>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::ContextMenuPolicy::DefaultContextMenu</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QDateEdit{
	background-color:rgb(255, 131, 133);
	border-radius:20px;
	padding-left:10px;
	color:black;

}</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::ButtonSymbols::NoButtons</enum>
        </property>
        <property name="displayFormat">
         <string>MM</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="amountBudget">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>330</y>
          <width>261</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Cascadia Code</family>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>IBeamCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QDoubleSpinBox{
	background-color:rgb(255, 131, 133);
	border-radius:20px;
	padding-left:10px;
	padding-right:10px;
	color:black;
}</string>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::ButtonSymbols::NoButtons</enum>
        </property>
        <property name="maximum">
         <double>1000000.000000000000000</double>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tableBack" native="true">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>30</y>
         <width>551</width>
         <height>481</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget{
	
	background-color:rgb(88, 88, 88);
	border-radius:10px;
}</string>
       </property>
       <widget class="QTableWidget" name="tableBudget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>501</width>
          <height>441</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
        </property>
        <column>
         <property name="text">
          <string>ID</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Category</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Month</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Amount</string>
         </property>
        </column>
       </widget>
      </widget>
      <zorder>tableBack</zorder>
      <zorder>Fback4</zorder>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
